tests/scipy_stats_test.py

```diff
@@ -564,6 +564,31 @@ def testIssue972(self):
       lsp_stats.norm.cdf(np.full((4,), np.inf, np.float32)),
       check_dtypes=False)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": jtu.format_test_name_suffix("", [shape], [dtype]),
```

Comment:
I thin this should be `[shape], dtype` because `dtype` is actually a tuple of dtypes.

This naming is relatively confusing, so perhaps below you could use `for (x_dtype, p_dtype) in itertools.product(...` and adjust the function signature accordingly.